src/components/logs/logsWebview.ts

```diff
@@ -22,11 +25,21 @@ export class LogsPanel extends WebPanel {
 
     public addContent(content: string) {
         this.content += content;
-        if (this.canProcessMessages) {
-            this.panel.webview.postMessage({
-                command: 'content',
-                text: content,
-            });
+        this.panel.webview.postMessage({
```

Comment:
Was the `canProcessMessages` check redundant?

Reply:
The `canProcessMessages` only checked if the panel was alive (= it was opened). It was useful because the childProcess didn't get killed so without this check the process would have kept writing in the panel which was already disposed and it would be resulted in an error. Because we now kill the process when the panel is disposed we don't need to check if it's "alive" or not.